wls-common/testing/src/main/java/de/muenchen/oss/wahllokalsystem/wls/common/testing/SecurityUtils.java

```diff
@@ -0,0 +1,32 @@
+package de.muenchen.oss.wahllokalsystem.wls.common.testing;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.provider.Arguments;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class SecurityUtils {
+
+    public static final String TESTUSER_DEFAULT = "TESTUSER";
+
+    public static final String TESTUSER_PASSWORD_DEFAULT = "TESTUSER_PASSWORD";
+
+    public static void runAs(String username, String password, String... authorities) {
+        SecurityContextHolder.getContext().setAuthentication(
+                new UsernamePasswordAuthenticationToken(username, password, AuthorityUtils.createAuthorityList(authorities)));
+    }
+
+    public static void runWith(String... authorities) {
+        SecurityUtils.runAs(TESTUSER_DEFAULT, TESTUSER_PASSWORD_DEFAULT, authorities);
+    }
+
+    public static Stream<Arguments> buildArgumentsForMissingAuthoritiesVariations(final String[] allRequiredAuthorities) {
```

Comment:
Zum Verständnis - dient diese Methode der Vereinfachung von Tests bei denen es getestet wird, ob beim Fehlen einer Authority die richtige Exception geworfen wird? Etwas wie das 
[wlsExceptionOnRepoWriteAuthorityMissing](https://github.com/it-at-m/Wahllokalsystem/blob/160-implementierung-er%C3%B6ffnungsuhrzeit/wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/eroeffnungsuhrzeit/EroeffnungsUhrzeitSecurityTest.java#L131)?
Und die Verwendung wäre so etwas wie 
`
SecurityUtils.buildVariantensForAuthoritiyList(alleAuthorities).get(index).get()
`
wobei Index der Index der Authority wäre die man ausschließen möchte ist? Ist "title" die Bezeichnung der Authority die man im Test ausschließen möchte?

Reply:
Könnte man die Verwendung ein bisschen beschreiben, als Kommentar?

Reply:
erledigt mit den commits 1a4774bbfaf662f24b46556d0caeae863bc447c7 und fd99acd6a936d67b7766c5cde89f7dcb6003a1c3